packages/compass-e2e-tests/helpers/commands/close-privacy-settings-modal.js

```diff
@@ -4,14 +4,30 @@ module.exports = function (app) {
   return async function closePrivacySettingsModal() {
     const { client } = app;
 
-    if (
-      !(await client.existsEventually(Selectors.PrivacySettingsModal, 5000))
-    ) {
+    // if (
+    //   !(await client.existsEventually(Selectors.PrivacySettingsModal, 5000))
+    // ) {
+    //   return;
+    // }
```

Comment:
Probably can be removed (here and in a few other places), right?

```suggestion
```

Reply:
Yup, ty had the commit on local and hadn't pushed up yet xD